hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestDataNodeMetrics.java

```diff
@@ -190,18 +190,28 @@ public Object answer(InvocationOnMock invocationOnMock)
       DataNodeFaultInjector.set(injector);
```

Comment:
And here we need to reserve original injector and reset it at finally scope.

Reply:
hi @ferhui Thank you for your comments!
It was revised as you suggested